org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel/src/org/eclipse/xtext/example/domainmodel/jvmmodel/DomainmodelJvmModelInferrer.xtend

```diff
@@ -32,8 +32,7 @@ class DomainmodelJvmModelInferrer extends AbstractModelInferrer {
 	def dispatch infer(Entity entity, extension IJvmDeclaredTypeAcceptor acceptor, boolean prelinkingPhase) {
 		accept(entity.toClass( entity.fullyQualifiedName )) [
 			documentation = entity.documentation
-			if (entity.superType !== null)
-				superTypes += entity.superType.cloneWithProxies
+			superClass = entity.superType
```

Comment:
@szarnekow just a small note: by using Xbase syntactic sugar, the use of the new API looks like above.
Is it better to be more explicit in this example and call `setSuperClass(entity.superType)`? We'd still use the implicit `it` anyway.

Reply:
I like it.

Reply:
I'm a fan as well, I just wanted to make sure :)

OK, so when the build is green I'll merge